Source/CesiumRuntime/Private/CesiumGltfComponent.cpp

```diff
@@ -1075,8 +1087,22 @@ static void loadPrimitive(
       }
     }
   } else {
-    TRACE_CPUPROFILER_EVENT_SCOPE(Cesium::ComputeFlatNormals)
-    computeFlatNormals(indices, StaticMeshBuildVertices);
+    if (material.hasExtension<ExtensionKhrMaterialsUnlit>()) {
+      glm::dvec3 ecefCenter = glm::dvec3(
+          transform * CesiumTransforms::unrealToOrFromCesium *
```

Comment:
I don't think the `unrealToOrFromCesium` here and below are needed. I tested by using Buffer Visualization -> World Normal and comparing the color of Melbourne with Cesium World Terrain. Cesium World Terrain will have some undulations in its normals, while Melbourne will be a solid color, but the overall color of them should be the same because the normals should point in basically up in both cases. With the `unrealToOrFromCesium` in place, they were quite different, but once I removed it they were the same. This makes sense because there's no need to use the Unreal coordinate system at all. The Origin is expressed in model coordinates, then we transform to ECEF, compute the normal, and transform the ECEF normal back to model coordinates.

There seems to be something weird with normals in Cesium for Unreal _in general_, though. Everything is good near the georeference origin, but on the back side of the globe something is wonky. For example, in this screenshot, both CWT and Melbourne are dark even though the sun is high in the sky:
![image](https://user-images.githubusercontent.com/924374/213166841-d2dc6fb4-2bb1-4e69-8afb-26981748b0fa.png)
This doesn't necessarily need to be fixed as part of this PR, but maybe you have an idea what's wrong? I'm at a bit of a loss at the moment.

Reply:
That makes sense, I assumed that there was code I couldn't find that automatically transformed the coordinates in the position view accessor.

Reply:
Could be precision related, but _in theory_ the directional light should just be a direction, so the large coordinate values shouldn't matter too much. But who knows. I'll write an issue.

Reply:
Oh! I think it's just this issue again:
https://github.com/CesiumGS/cesium-unreal/issues/527
And there's a workaround listed in there.